CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+- Add code-highlighting to tutorials (#108, by @patricoferris)
+- Add initial set of Best Known Methods (#107, by @tmattio)
+- Add initial toplevel to homepage (#106, by @tmattio)
+- Serve HTML pages from server (#101, by @tmattio)
```

Comment:
This is not a user-facing change, so I'm not sure it should be included in the changelog (same for the previous items "Merge frontend" and "Merge ood")